@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -491,11 +491,11 @@ export class PoiManager {
             const fadeNear =
                 technique.fadeNear !== undefined
                     ? getPropertyValue(technique.fadeNear, displayZoomLevel)
-                    : 0;
+                    : technique.fadeNear;
             const fadeFar =
                 technique.fadeFar !== undefined
                     ? getPropertyValue(technique.fadeFar, displayZoomLevel)
-                    : 0;
+                    : technique.fadeFar;
```

Comment:
Similarly to above, but by default I would assign 1.0 or defaultRenderParams.fadeFar.